src/common/utils/path-utils.ts

```diff
@@ -0,0 +1,8 @@
+import * as path from "path";
+
+export function pathStartsWith(one: string, another: string): boolean {
+  return one.startsWith(another) && (
```

Comment:
Use of the plain startsWith method lead us to `/a/path/.gitt` path to be matched against the `/a/path/.git` template and then ignored. Such a false-positive matching is incorrect, hence this method was born.